app/manage/lib/core/screens/user_profile_screen.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:manage/core/model/user_profile_model.dart';
+
+
+class UserPage extends StatelessWidget {
+
+  final UserProfileModel user;
+
+  UserPage({this.user});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      extendBodyBehindAppBar: true,
+      appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        elevation: 0,
+        iconTheme: IconThemeData(color: Colors.black),
+        actions: [
+          IconButton(
+            icon: Icon(Icons.do_not_disturb),
+            onPressed: (){
```

Comment:
You could implement this logic by how login works. You can check this by looking at login request at user_service.dart . You can also see how routes are delegated, like how pages is implemented for authentication logic.